models/token.js

```diff
@@ -9,12 +9,9 @@ const MODEL = {
         .integer()
         .positive(),
 
-    uuid: Joi
+    value: Joi
         .string()
```

Comment:
The `model` will use it for saving to the datastore.

It's purely coincidence that secret values don't care about content or length. Because of that, you could use the same criteria for verifying the sealed and unsealed versions. 

However, you won't have that same guarantee with the token model. This restriction can be kept loose. Which means we can't guarantee it will be the form we say it should be (40 characters longs, etc).